### ISSUE TITLE
No carga la pagina completa de https://developer-mozilla-org.translate.goog/en-US/docs/Web/API/Navigator/clipboard?_x_tr_sl=auto&_x_tr_tl=es&_x_tr_hl=es-419

### ISSUE THREAD

**Issue by Os-688** [mdn/content#23868]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard

### What specific section or headline is this issue about?

no aparece  contenido en la pantalla. No carga.

### What information was incorrect, unhelpful, or incomplete?

*

### What did you expect to see?

la información completa.

### Do you have any supporting links, references, or citations?

https://developer-mozilla-org.translate.goog/en-US/docs/Web/API/Navigator/clipboard?_x_tr_sl=auto&_x_tr_tl=es&_x_tr_hl=es-419

### Do you have anything more you want to share?

*

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/navigator/clipboard`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/navigator/clipboard/index.md
* Last commit: https://github.com/mdn/content/commit/4b4638246aad5d39b9a2e5c572b179b4c39c0a84
* Document last modified: 2022-09-09T05:17:24.000Z

</details>

**Comment by bsmth** [comment]
Hi @Os-688, I don't really understand what you expect to see here and the links you have provided work okay for me. Note that we can't guarantee how page functionality works when browsing via translate.google.com. Could you provide some more information?

**Comment by dawei-wang** [comment]
Hi @bsmth, the BCD table doesn't load when translated. This may be what @Os-688 is referring to.

![BCD English](https://user-images.githubusercontent.com/11358567/215006799-15ad8903-1472-441e-8fb9-bf512009e87e.png)

![BCD translated](https://user-images.githubusercontent.com/11358567/215006906-b906f066-758e-4260-926c-6ad42a265fa1.png)

**Comment by bsmth** [comment]
You're right @dawei-wang, thanks for checking. I would point to the `es` version of this page from https://developer.mozilla.org/es/docs/Web/API/Navigator, but we don't have a sub-page for clipboard yet.
